cmake/FindPVRTexLib.cmake

```diff
@@ -22,7 +22,8 @@
 #  PVRTEXLIB_LIBRARIES - The libraries needed to use PVRTexLib
 
 IF (${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
-  SET( PVRTEXLIB_ROOT "/Applications/Imagination/PowerVR_Graphics/PowerVR_Tools/PVRTexTool/Library" )
+  SET( PVRTEXLIB_ROOT "/Applications/Imagination/PowerVR_Graphics/PowerVR_Tools/PVRTexTool/Library"
```

Comment:
This option should be documented in README.md under the "Miscellaneous Options" section.